Sources/BrowserServicesKit/Autofill/WebsiteAutofillUserScript.swift

```diff
@@ -114,14 +114,14 @@ public class WebsiteAutofillUserScript: AutofillUserScript {
         var response = GetSelectedCredentialsResponse(type: CredentialsResponse.none)
 
         let emailSignedIn = emailDelegate?.autofillUserScriptDidRequestSignedInStatus(self) ?? false
-        if (previousEmailSignedIn == nil) {
+        if previousEmailSignedIn == nil {
             previousEmailSignedIn = emailSignedIn
         }
         let hasEmailSignedInStateChanged = previousEmailSignedIn != emailSignedIn
         let inContextEmailSignupPromptDismissedPermanentlyAt: Double? = emailDelegate?.autofillUserScriptDidRequestInContextPromptValue(self)
         let hasIncontextSignupStateChanged = previousIncontextSignupPermanentlyDismissedAt != inContextEmailSignupPromptDismissedPermanentlyAt
 
-        if (hasEmailSignedInStateChanged || hasIncontextSignupStateChanged) {
+        if hasEmailSignedInStateChanged || hasIncontextSignupStateChanged {
```

Comment:
Honestly maybe the same here, I feel like it's the rule for brackets for the overall if statement not being sophisticated enough to know when there is more than one clause. and whenever there's more than one clause, I feel the same way about ifs as I do with maths: brackets are almost always good

Reply:
I think I'd agree here too. I'm inclined to go ahead with this change for now since it gets us into compliance with the current rules but perhaps have a discussion later about how we want to change the rules in the future.